### PR TITLE
feat Added landing page

### PR DESCRIPTION
**Pull request by ayush65**
Screenshot (103)
Screenshot (104)

### CODE REVIEW

**Review by shraddha15-creator — APPROVED** [review]
Looks good!

**Review by shubhambishnoi — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (jeetsdev) [inline .vscode/settings.json:3]: Add this file in .gitignore

**Review by jeetsdev — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+@import url("https://angry-bird-ui.netlify.app/components/All-css-files/main.css");
+
```
Comment (abhisekgupta001) [inline style.css:2]: remove the margin from body, use reset here.

**Review by anjali1102 — APPROVED** [review]

File: style.css (hunk 2 above)
Comment (ayush65) [reply style.css:3]: yea i have done that . Thanks for reviewing

File: .vscode/settings.json (hunk 1 above)
Comment (ayush65) [reply .vscode/settings.json]: I have done that thanks for the suggestion . Thanks for reviewing.

**Review by NtshVrm — APPROVED** [review]

**Review by abhisekgupta001 — APPROVED** [review]

**Pull request merged**